### ISSUE TITLE
verify that `extendedMethods` only contain newly created extension methods ... otherwise initialization logic is incorrect

### ISSUE THREAD

**Issue by dalehenrich** [dalehenrich/Rowan#41]
The rule for running initializers is that all new or changed class-side methods with the `#initialize` selector should be executed with the following method:
:
```smalltalk
runInitializers

	"run the class initialization methods as needed"

	(addedMethods copy
		addAll: extendedMethods;
		addAll: methodsNeedingRecompile;
		yourself) do: [ :methodPatch | methodPatch runInitializer ]
```
I believe that `extendedMethods`is exclusively used for added and changed extensions methods, but we need tests to validate that the initializations are handled correctly. Note that we need to include `#initialize` methods being moved between packages in the test case to make sure that moves are excluded..

**Comment by dalehenrich** [comment]
cleaned up initialization logic ... at end of day, there were two bugs fixed:
1. initialize method run when method changed interactively (fixed)
2. initialize method incorrectly run when initialize method moved without change between packages

**Comment by dalehenrich** [comment]
fixed with pr #112
